src/qml/controls/OnboardingInfo.qml

```diff
@@ -9,8 +9,12 @@ import org.bitcoincore.qt 1.0
 
 Item {
     id: root
-    property alias banner: banner_loader.sourceComponent
+    property alias bannerItem: banner_loader.sourceComponent
+    property alias detailItem: detail_loader.sourceComponent
     required property string buttonText
+    property bool banner: true
+    property bool detail: false
+    property bool finish: false
```

Comment:
I think these three bools are too vague. Maybe "bannerActive", "detailActive", and "finalItem" or "lastPage".  I also think "detail" isn't great but I can't really think of anything better so I don't know that there is an alternative there that would clarify the intent.

Reply:
addressed in [e3b5e2a](https://github.com/bitcoin-core/gui-qml/commit/e3b5e2aca6b04eba386d419e5b18da7d32c09953), thanks!